client/src/components/feedback/comment.js

```diff
@@ -1,15 +1,21 @@
 import React, { Component } from 'react';
-import { Container, Image } from 'react-bootstrap';
+import {
+  Container, Image, Badge,
+} from 'react-bootstrap';
 import '../../styles/feedback-comment.css';
 import PropTypes from 'prop-types';
 import Api from '../../services/api';
+import Person from '../../services/person';
```

Comment:
Don't use this class. I've removed it in https://github.com/Social-projects-Rivne/Lunch_Time/pull/109
You can write similar code, as in my PR, or wait to merge it in dev and update your code

Reply:
Ok, let your PR be merged first. Then I will merge with dev.